src/community/advanced-data-card.tsx

```diff
@@ -0,0 +1,462 @@
+import * as React from 'react';
+import {Boxed} from '../boxed';
+import {sprinkles} from '../sprinkles.css';
+import Stack from '../stack';
+import * as styles from './advanced-data-card.css';
+import Divider from '../divider';
+import {Text2, Text4} from '../text';
+import {useTheme} from '../hooks';
+import {vars} from '../skins/skin-contract.css';
+import IconCloseRegular from '../generated/mistica-icons/icon-close-regular';
+import IconButton from '../icon-button';
+import Inline from '../inline';
+import Box from '../box';
+import Touchable from '../touchable';
+import classNames from 'classnames';
+
+import type StackingGroup from '../stacking-group';
+import type Image from '../image';
+import type {ButtonPrimary, ButtonLink} from '../button';
+import type {DataAttributes, IconProps} from '../utils/types';
+import type {RendersNullableElement} from '../utils/renders-element';
+import type Tag from '../tag';
+
+type CardAction = {
+    label: string;
+    onPress: () => void;
+    Icon?: React.FC<IconProps>;
+    iconSize?: number;
+    iconColor?: string;
+    iconBackground?: string;
+    iconBackgroundInverse?: string;
+};
+
+type CardActionsGroupProps = {
+    actions: Array<CardAction>;
+    isInverse?: boolean;
+};
+
+const CardActionsGroup = ({actions, isInverse}: CardActionsGroupProps): JSX.Element => {
```

Comment:
Please, remove this component and use the one in `card.tsx` instead. You can make it public in `cad.tsx` and import it here.